code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -301,18 +349,28 @@ func (sng StorageNodeGeolocation) validate() error {
 	return nil
 }
 
+type partitionLocation struct {
```

Comment:
I guess this is for blobber partition location only, if so, please do not use the common name (partition location) for it, rename it to properly such as `RewardPartitionLocation`.